docs.css

```diff
@@ -1,38 +1,63 @@
-@import url('style.css');
-
-.doc-container{
-    display: flex;
-
-}
-main{
-    padding: 1rem;
-}
-section{
-    min-height: 50vh;
-    border: 2px solid var(--color-primary);
-    padding: 1rem;
-    margin: 1rem;
-}
-.navbar{
-    position: sticky;
-    top: 0;
-}
-.sidebar{
-    background-color: var(--color-background);
-    flex-basis: 500px;
-    position: sticky; 
-    top: 4rem;
-    left: 0;
-    height: calc(100vh - 4rem);
-    overflow: auto;
-}
-.sidebar-list{
-    padding: 1rem;
-    display: flex;
-    flex-direction: column;
-    gap: 0.5rem;
-    align-items: center;
-}
-.sidebar-list-item{
-    color: var(--color-primary);
-}
\ No newline at end of file
+@import url("style.css");
+:root {
+  --color-font: #333;
+}
+.doc-container {
+  display: flex;
+}
+main {
+  padding: 1rem;
+}
```

Comment:
you should avoid giving styles to html elements directly. you can use class or id for it